src/package.json

```diff
@@ -0,0 +1,154 @@
+{
```

Comment:
is this package.json superfluous?

Reply:
my bad, removed

Reply:
This seems not right - don't rollback package,json